### ISSUE TITLE
No file encoding button display

### ISSUE THREAD

**Issue by xiaoyun94** [cdr/code-server#1437]
VS code has a button at the bottom to switch file encode type, such as gbk, utf8, but it does not show up tn code server. i have tried in my os deepin 15.10, open suse 15.1, suse server 12 sp3, all of them were missing this func. but i found ur shortcuts have this button, so i am wondering why this func is missing in my OSs.

**Comment by xiaoyun94** [comment]
V3 matters.
V1 works well

**Comment by kylecarbs** [comment]
Thanks for the report. We should definitely support multiple file-encoding types.

**Comment by gnouts** [comment]
This is a blocker for me.
If I try to use v1 I'm stuck with this bug https://github.com/cdr/code-server/issues/338
And v2 and v3 doesn't offer to change encoding.
I try changing the settings.json with `"files.encoding": "windows1252"` but it doesn't seem to do anything.
Is there a workaround to change encoding ? (I really need to open files with special char like ¤ )

**Comment by code-asher** [comment]
https://github.com/microsoft/vscode/issues/79275

**Comment by kylecarbs** [comment]
@gnouts sorry to hear. We can definitely fix this if enough interest arises. For now, unfortunately it's a pretty low-priority issue.

Feel free to take a stab at it thought, we'd happily merge your contribution :)

**Comment by xiaoyun94** [comment]
a sad story for Chinese developers

**Comment by Shagrat2** [comment]
UTF-8 open as Layout U.S. and kill not english  chars

**Comment by gyzerok** [comment]
I'd like to contribute to this issue by specifying my use-case, so you can keep it in mind once it gets unblocked.

I need to be able to specify encoding on a per-language basis.

**Comment by nhooyr** [comment]
Duplicate of #1017
